include/libParamTypes.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: ODbL-1.0
+
+PARAM_TYPE_32(unk) // Unknown, currently intended to be used internally. May be subject to change accessible to public usage.
```

Comment:
```suggestion
PARAM_TYPE_32(unk) // Unknown, currently intended to only be used internally. This may change in the future (e.g. it may become accessible to the user).
```

Reply:
See https://github.com/Cxbx-Reloaded/XbSymbolDatabase/pull/208#discussion_r1685670653

Closing as whole comment will be replace with entirely different comment.